autoload/utils/cmake/fileapi.vim

```diff
@@ -34,10 +34,26 @@ function! s:createQuery() abort
     return l:query
 endfunction
 
-function! s:parseCodemodel(path, hash) abort
-    let l:codemodel = json_decode(join(readfile(a:path), ''))
+function! s:parseTarget(path) abort
+    let l:targetInfo = json_decode(join(readfile(a:path), ''))
+    let l:pathes = []
+    for l:artifact in l:targetInfo['artifacts']
+        call add(l:pathes, l:artifact['path'])
+    endfor
+    return {'type': l:targetInfo['type'], 'pathes': l:pathes}
```

Comment:
I am not sure that it is a real case when we have several artefacts and target type is executable.

Reply:
in this function, general info about targets is extracted, so later on patches can be used not only for executables

Reply:
Also, I have found out, that on Windows there are actually two artefacts (exe and pdb files)

Reply:
Cool, I didn't know it.